useless-manager/docs/help.txt

```diff
@@ -0,0 +1,30 @@
+-  manager.py is licensed under GNU General Public License v3.0
```

Comment:
https://github.com/TheAlgorithms/Python/blob/master/LICENSE.md is the license for all code in this repo.  If you do not agree with the MIT license then please close this pull request.